### DISCUSSION TITLE
Erro ao executar o dotNet run BaltaDataAccess -  Curso Acesso à dados com .NET, C#, Dapper e SQL Server

### DISCUSSION THREAD

**Discussion by paulinhosimao** [balta-io/forum#316] [Tira Dúvidas] [answered]
Boa noite professor, náo consegui executar o exemplo dotNet run BaltaDataAccess , segue abaixo meus dados do BD local e não vai de jeito algum.

![MyConLocalBD](https://user-images.githubusercontent.com/19830052/234131223-4812cdc4-05f8-46b8-aae8-259d30cfb115.png)
![ERROR_CONNECT_BD_LOCAL](https://user-images.githubusercontent.com/19830052/234131228-4e6d9394-a632-496c-bad7-e21e672c7b20.png)

**Comment by paulinhosimao** [comment, marked as answer]
Professor @andrebaltieri , encontrei o erro, estava na chamada do servidor, tinha que colocar duas barras \\.

**Reply by andrebaltieri** [reply]
🚀

**Comment by Lehzinho** [comment]
Estou tendo o mesmo problema, ja fui no git da aula e nao consigo ver o erro.
Poderia me dar uma ajuda por favor?
![Connection](https://user-images.githubusercontent.com/99737712/236303391-a289ffb8-9dd3-4527-858a-5d1eac12810c.JPG)
![Code](https://user-images.githubusercontent.com/99737712/236303395-1eb3c9a3-6b1d-43c0-bd6b-1fbd16323143.JPG)
![Error](https://user-images.githubusercontent.com/99737712/236303402-4008ed1d-232b-4c8b-84ff-a5f835674cba.JPG)

**Reply by Lehzinho** [reply]
Resolvido a versao do Microsoft.Data.SqlClient que eu estava usando estava bem acima do que o da aula, fiz o download da versao=2.1.3 e funcionou.

O vídeo dessa aula pode estar desatualizado. Seria possível considerar fazer uma atualização do conteúdo para que os alunos tenham acesso às informações mais recentes? Acredito que isso ajudaria a melhorar ainda mais a qualidade do seu conteúdo. Obrigado pela atenção.
